sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/LastEnqueuedEventProperties.cs

```diff
@@ -66,6 +95,7 @@ public struct LastEnqueuedEventProperties : IEquatable<LastEnqueuedEventProperti
         internal LastEnqueuedEventProperties(EventData sourceEvent) :
             this(sourceEvent?.LastPartitionSequenceNumber,
                  sourceEvent?.LastPartitionOffset,
+                 sourceEvent?.ReplicationSegment,
```

Comment:
TODO: According to the service doc, `"last_enqueued_sequence_number_epoch"` is only part of the response map and is **not** exposed on the amqp message properties. I don't think that this approach will work for this scenario? I need to investigate more, but that basically would only allow us to return the true last enqueued replication segment as part of the `PartitionProperties` and not on the `LastEnqueuedEventProperties`?

Reply:
Maybe that's fine, maybe we just don't return the `ReplicationSegment` on the `LastEnqueuedEventProperties` at all? Since there's an easy default for "latest" replication segment ("-1" or "")?